tests/unit/records/test_records_receivers.py

```diff
@@ -598,3 +600,111 @@ def test_populate_abstract_source_suggest_does_nothing_if_record_is_not_hep():
         {'source': 'foo'},
         {'source': 'bar'},
     ]
+
+
+def test_populate_affiliation_suggest():
```

Comment:
This test is **really** hard to follow. I will split it in several, each dealing with a different field. This is not just for readability: I will also add tests to ensure that the schema is what is asserted by the code, because the `institutions` schema is going to change soon, so it would be nice to have good errors on this.